src/.eslintrc.js

```diff
@@ -63,7 +63,7 @@ module.exports = {
           },
           {
             pattern:
-              "(analytics|components|constants|context|gql|hoc|hooks|pages|types|utils)/**",
+              "{App,analytics,components,constants,context,gql,hoc,hooks,pages,test_utils,types,utils}{,/**}",
```

Comment:
It would probably make more sense to remove `App` here and include it as a relative import in src/index.tsx
```js
import App from "./App"
```
```suggestion
              "{analytics,components,constants,context,gql,hoc,hooks,pages,test_utils,types,utils}{,/**}",
```